smack-core/src/testFixtures/java/org/jivesoftware/smack/test/util/WaitForPacketListener.java

```diff
@@ -43,7 +43,7 @@ public void waitAndReset() {
 
     public void waitUntilInvocationOrTimeout() {
         try {
-            boolean res = latch.await(300, TimeUnit.SECONDS);
+            boolean res = latch.await(10, TimeUnit.SECONDS);
```

Comment:
I don't see a reason why this should be lowered to 10 seconds. Especially since 10 seconds is easily in the area where heavily congested systems may delay a thread. I'd like to prefer the original value, we could, however talk about making this configurable. But then, as above, no as part of this PR.